chapter2.adoc

```diff
@@ -44,41 +44,29 @@ When `mdbgen` is set to 0, the <<dbops, debug operations>> are disallowed and th
 ==== Supervisor Domain Debug Control
 The Smsdedbg extension cite:[smmtt] introduces `sdedbgalw` field (bit 7) in CSR <<Sdseccsr,msdcfg>>, to control the debuggability of supervisor domains. When `mdbgen`=0, the `sdedbgalw` field determines both the debug allowed privilege modes and the <<dbgaccpriv, debug access privilege>>, as illustrated in <<dbgpriv>>. 
 
+[[dbgaccpriv]]
+==== Debug Access Privilege
+The *debug access privilege* is defined as the privilege level granted to the external debugger when performing state accesses via the hart, such as Abstract Commands and Program Buffer execution. Memory and register accesses from Debug Mode carry *debug access privilege* instead of always with M-mode, while the privilege is determined by <<mdbgctl, `mdbgen`>> and <<submdbgctl, `sdedbgalw`>>, as listed in <<dbgpriv>>. Illegal access will return an error and set `abstractcs.cmderr` to 3.
+
+The `dmprv` field in `dcsr` modifies the effective *debug access privilege* to memory. When `dmprv`=0, the *debug access privilege* to memory follows <<dbgpriv>>; when `dmprv`=1, the effective *debug access privilege* to memory is represented by `prv` and `v` fields in `dcsr`. 
+
+[NOTE]
+When `dmprv`=1, the debugger memory access privilege matches the hart's privilege at Debug Mode entry. With `dmprv`, the debugger may modify the memory access privilege to VS/VU/U in Debug Mode when `mdbgen`=0 and `sdedbgalw`=1. However, the `dmprv` is still a debug feature, and it may be proposed to RISC-V Debug specification cite:[dbgspec] in the future to facilitate specific debug use cases. 
+
```

Comment:
I would remove lines 51 and 54 above.  Let's keep the definition of `dmprv` in one place, and no need to speculate about the debug spec.